Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -450,6 +545,106 @@ public extension TransposedConv2D {
     }
 }
 
+
+/// A 3-D transposed convolution layer (e.g. spatial transposed convolution over images).
+///
+/// This layer creates a convolution filter that is transpose-convolved with the layer input
+/// to produce a tensor of outputs.
+@frozen
+public struct TransposedConv3D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The 5-D convolution kernel.
+    public var filter: Tensor<Scalar>
+    /// The bias vector.
+    public var bias: Tensor<Scalar>
+    /// The element-wise activation function.
+    @noDerivative public let activation: Activation
+    /// The strides of the sliding window for spatial dimensions.
+    @noDerivative public let strides: (Int, Int, Int)
+    /// The padding algorithm for convolution.
+    @noDerivative public let padding: Padding
+    @noDerivative public let paddingIndex: Int
```

Comment:
Add a doc comment for this property.

Reply:
@rxwei this property isn't really exposed to the users, it's a temporary property which allows us to handle computation based on what kind of padding it is. we currently support `.same` and `.valid`.
Do we still document this?